# SwarmCommand.msg

string agent
uint8 type
uint32 order
uint32 data_length
uint8[] data
